index.html

```diff
@@ -0,0 +1,232 @@
+				</p>
+				<div class="ava">Available</div>
+			</section>
+			<section class="specSec">
+				<header class="secHead">Specification</header>
+				<div class="specs">
+					Size= 4" x 2"
+				</div>
+				<div class="specs">
+					Weight= 2oz
+				</div>
+				<footer>
+					Specifications valid until 4/5/1017
+				</footer>
+			</section>
+			<section class="price">
+				<header class="priceHead">Pricing</header>
+				    <!-- 1. Each block element contains information about the price for different quantities. -->
+
+				<div class="quant">1-10: $1.99</div>
+
+				<div class="quant">11-20: $.89</div>
+
+				<div class="quant">21-30: $.60</div>
+
+			</section>
+		</article>
+	<!-- THIS IS CARD 5 -->
+		<article class="card">
+			<header>
+				<h1>Bell Pepper</h1>
+			</header>
+			<section class="proDes">
+				<div><img src="img/bell-pepper.jpg" alt=""></div>
+				<p>
+					Soko radicchio bunya nuts gram dulse silver beet parsnip napa cabbage lotus root sea lettuce brussels sprout cabbage. Catsear cauliflower garbanzo yarrow salsify chicory garlic bell pepper napa cabbage lettuce tomato.
+				</p>
+				<div class="ava">Available</div>
+			</section>
+			<section class="specSec">
+				<header class="secHead">Specification</header>
+				<div class="specs">
+					Size= 4" x 2"
+				</div>
+				<div class="specs">
+					Weight= 2oz
+				</div>
+				<footer>
+					Specifications valid until 4/5/1017
+				</footer>
+			</section>
+			<section class="price">
+				<header class="priceHead">Pricing</header>
+				    <!-- 1. Each block element contains information about the price for different quantities. -->
+
+				<div class="quant">1-10: $1.99</div>
+
+				<div class="quant">11-20: $.89</div>
+
+				<div class="quant">21-30: $.60</div>
+
+			</section>
+		</article>
+	<!-- THIS IS CARD 6 -->
+		<article class="card">
+			<header>
+				<h1>Eggplant</h1>
+			</header>
+			<section class="proDes">
+				<div class="img"><img src="img/Eggplant-5.png" alt=""></div>
+				<p>
+					JÃ­cama garlic courgette coriander radicchio plantain scallion cauliflower fava bean desert raisin spring onion chicory bunya nuts. Sea lettuce water spinach gram fava bean leek dandelion silver beet eggplant bush tomato.
+				</p>
+				<div class="ava">Available</div>
+			</section>
+			<section class="specSec">
+				<header class="secHead">Specification</header>
+				<div class="specs">
+					Size= 4" x 2"6inx4in
+				</div>
+				<div class="specs">
+					Weight= 0.3lb 2oz
+				</div>
+				<footer>
+					Specifications valid until 4/5/1017
+				</footer>
+			</section>
+			<section class="price">
+				<header class="priceHead">Pricing</header>
+				    <!-- 1. Each block element contains information about the price for different quantities. -->
+
+				<div class="quant">1-10: $1.99</div>
+
+				<div class="quant">11-20: $.89</div>
+
+				<div class="quant">21-30: $.60</div>
+
+			</section>
+		</article>
+
+</body>
+</html>
```

Comment:
As Github helpfully points out with its 🚫  symbol, putting a blank line at the end of files is convention, a holdover from þe olde POSIX days.  If you'd like to know more, you can read about it [here](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).